WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrderViewModel.swift

```diff
@@ -2,16 +2,51 @@ import Yosemite
 
 /// View model for `NewOrder`.
 ///
-final class NewOrderViewModel {
+final class NewOrderViewModel: ObservableObject {
+    private let siteID: Int64
+    private let stores: StoresManager
+
     /// Order to create remotely
     ///
     private var order: Order = .empty {
         didSet {
-            isCreateButtonEnabled = true
+            navigationTrailingItem = .create(rendered: false)
```

Comment:
Shouldn't this be `create(rendered: true)`.
I find this `create` button situation a bit complicated to understand from reading the code because it starts as invisible and then it's visible after making the first request. 

Wouldn't it make more sense to start as visible and then hide it after the request succeeds? 
If not I think adding some comments or TODO around it would help a ton!

What do you think?

Reply:
> Shouldn't this be create(rendered: true).

Yep, I mixed up what I was trying to do here; thanks for catching it!

> I find this create button situation a bit complicated to understand from reading the code because it starts as invisible and then it's visible after making the first request.

In the designs, when you first navigate to this screen there is no `create` button. The `create` button only appears once you start adding things to the order (e.g. products or customer info). My thought was to update the `order` var as things are added to the order in the UI, and then send `order` to the remote endpoint to create the order remotely.

I can make that clearer in the comments but I can also make that clearer by explicitly enabling the `create` button when `order` isn't empty (instead of just enabling it when `order` is changed). I'll work on that change.